mappings/sweet-envo_cryosphere.sssom.tsv

```diff
@@ -0,0 +1,199 @@
+#this mapping was created following and expanding upon the SSSOM guidance: https://mapping-commons.github.io/sssom/spec/
+#dcterms:license: CC-0
+#version: 0.1
+#versions
+#  - ENVO: v2021-05-14
+#  - SWEET: v3.5.0
+#creator_id:
+#  - orcid:0000-0003-4808-4736
+#  - orcid:0000-0002-4366-3088
+#  - orcid:0000-0002-8454-8481
+#  - orcid:0000-0002-3410-4655
+#  - orcid:0000-0001-7543-6173
+#creator_name:
+#  - Ruth Duerr
+#  - Pier Luigi Buttigieg
+#  - Kate Rose
+#  - Kai Blumberg
+#  - Nancy Wiegand
+#curie_map:
+#  ENVO: "http://purl.obolibrary.org/obo/ENVO_"
+#  soreas: "http://sweetontology.net/realmSoil/"
+#  sophatmow: "http://sweetontology.net/phenAtmoWind/"
+#  soreac: "http://sweetontology.net/realmCryo/"
+#  sophhy: "http://sweetontology.net/phenHydro/"
+#  sorealg: "http://sweetontology.net/realmLandGlacial/"
+#  sophgg: "http://sweetontology.net/phenGeolGeomorphology/"
+#  sophatmopc: "http://sweetontology.net/phenAtmoPrecipitation/"
+#  somaw: "http://sweetontology.net/matrWater/"
+#  sophcr: "http://sweetontology.net/phenCryo/"
+#  sophatmofr: "http://sweetontology.net/phenAtmoFront/"
+#  soprocsc: "http://sweetontology.net/procStateChange/"
+#  sophso: "http://sweetontology.net/phenSolid/"
+#  sorea: "http://sweetontology.net/realm/"
+#  soreahb: "http://sweetontology.net/realmHydroBody/"
+#  sophatmofo: "http://sweetontology.net/phenAtmoFog/"
+#  sophgf: "http://sweetontology.net/phenGeolFault/"
+#  sorealo: "http://sweetontology.net/realmLandOrographic/"
+#  sophatmowm: "http://sweetontology.net/phenAtmoWindMesoscale/"
+#  soma: "http://sweetontology.net/matr/"
+#  sophsy: "http://sweetontology.net/phenSystem/"
+#  sophfd: "http://sweetontology.net/phenFluidDynamics/"
+#  sophft: "http://sweetontology.net/phenFluidTransport/"
+#  sorepsg: "http://sweetontology.net/reprSpaceGeometry/"
+#  sopropst: "http://sweetontology.net/propSpaceThickness/"
+#  soreacz: "http://sweetontology.net/realmClimateZone/"
+#  orcid: "https://orcid.org/"
+#  skos: "http://www.w3.org/2004/02/skos/core#"
+subject_id	subject_label	object_id	object_label	predicate_id	match_type	Comments
+ENVO:01001610	active layer in a cryosphere	soreas:ActiveLayer	active layer	skos:closeMatch	HumanCurated	SWEET does not have a definition.  This could change to exactMatch if SWEET adopted the ENVO definition.

```

Comment:
```suggestion
ENVO:01001610	active layer in a cryosphere	soreas:ActiveLayer	active layer	skos:closeMatch	semapv:ManualMappingCuration	SWEET does not have a definition.  This could change to exactMatch if SWEET adopted the ENVO definition.
```

Reply:
Make that change in all rows! You have been using an old SSSOM spec.